napari/utils/colormaps/colormap_utils.py

```diff
@@ -61,7 +61,12 @@
 _MATPLOTLIB_COLORMAP_NAMES_REVERSE = {
     v: k for k, v in matplotlib_colormaps.items()
 }
-_VISPY_COLORMAPS_ORIGINAL = _VCO = get_colormaps()
+
+# some colormaps use BaseColormap and custom mapping functions instead of
+# standard colors/controls, so they are broken in napari
```

Comment:
Whoa, like what, and is that something we should support? Could be handy. 😃

Reply:
Not really; basically, you can create a `BaseColormap` and define your own glsl function for it, which might be whatever crazy thing you want. However, it's not really easy to communicate between python and "stringy glsl", so I don't think it's a good idea.